server/src/immich/api-v1/asset/asset.service.ts

```diff
@@ -571,18 +572,25 @@ export class AssetService {
   }
 
   private getThumbnailPath(asset: AssetEntity, format: GetAssetThumbnailFormatEnum) {
+    if (!Object.values(GetAssetThumbnailFormatEnum).includes(format)) {
+      throw new UnsupportedMediaTypeException(`Unsupported thumbnail format requested: ${format}`);
+    }
+
     switch (format) {
       case GetAssetThumbnailFormatEnum.WEBP:
-        if (asset.webpPath && asset.webpPath.length > 0) {
-          return asset.webpPath;
+        if (asset.webpPath) {
+          return [asset.webpPath, GetAssetThumbnailFormatEnum.WEBP];
```

Comment:
Can we just pass a valid content type here? No need to get fancy I think. Just return `image/webp`

Reply:
Sure thing